.github/workflows/manual-nixpkgs.yml

```diff
@@ -13,14 +12,8 @@ jobs:
   nixpkgs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-        with:
-          # pull_request_target checks out the base branch by default
-          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+      - uses: actions/checkout@v2.3.4
       - uses: cachix/install-nix-action@v13
-        with:
-          # explicitly enable sandbox
-          extra_nix_config: sandbox = true
```

Comment:
Why delete this?

Reply:
It's the default value, so it provides no benefit. It would also break macOS in case we'd enable it.